### PR TITLE
#731 Using native webshare API for sharing

### LINKED ISSUE #731: Use native webshare API on mobiles
**Is your feature request related to a problem? Please describe.**
Current share menu is not so nice on mobile

**Describe the solution you'd like**
.it would be nice to use webshare api

https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share

https://css-tricks.com/how-to-use-the-web-share-api/

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the KodaDot NFT gallery.
Let's do a quick check before the merge.
Your contribution will be an eternal part of the community codebase.

### PR type
- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### Before submitting this PR, please make sure:
- [x] Code builds clean without any errors or warnings
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

### Optional
- [x] I've tested it on mobile and everything works
- [ ] I found edge cases

### What's new? (may be part of changelog)
- This PR closes #731 
Have used share api for all the platforms where its supported

### If issue had bounty label ($,$$,$$$)
- Fill up your KSM address: 
https://kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU

### Community participation
- [x] Are you at KodaDot Discord? If not yet, [let's join us - https://discord.gg/35hzy2dXXh](https://discord.gg/35hzy2dXXh)

### Screenshot
- [x] Whenever my fix has changed **something** on UI, a screenshot for others, is more than welcome
WhatsApp Image 2021-09-25 at 8 48 07 PM

### CODE REVIEW

**Comment by yangwao** [comment]
Seems working well! 
Let me wait for @roiLeo review if everything is okay
telegram-cloud-photo-size-4-5943001867757925936-y

**Review by roiLeo — CHANGES_REQUESTED** [review]
LGTM works well on mobile, just a quick change to fix linting errors.

File: src/components/rmrk/Gallery/Item/Sharing.vue [hunk 1]
```diff
@@ -229,6 +229,29 @@ export default class Sharing extends Vue {
   public toast(message: string): void {
     this.$buefy.toast.open(message);
   }
+
+  public async shareTooltip() {
+    this.openFallbackShareTooltip();
+    if (navigator.share) {
+      const shareData = {
+        title: "KodaDot",
```
[the remark this answers is not in the document]
Comment (prachi00) [reply src/components/rmrk/Gallery/Item/Sharing.vue]: fixed

**Review by roiLeo — APPROVED** [review]

**Review by vikiival — APPROVED** [review]
WFM

**Pull request merged**